src/engine/enginedeck.cpp

```diff
@@ -43,6 +44,11 @@ EngineDeck::EngineDeck(const ChannelHandleAndGroup& handle_group,
         pEffectsManager->registerChannel(handle_group);
     }
 
+    // Make input_configured read-only.
+    m_pInputConfigured->connectValueChangeRequest(
+        this, SLOT(slotInputConfiguredChangeRequest(double)),
+        Qt::DirectConnection);
```

Comment:
Yea, in engine code we largely don't do things how Qt would suggest. In this case, we're disabling changes to the CO by connecting the change request to an empty slot. If this were an AutoConnection, it would take a round trip through the main thread event loop -- when all we really want is to dump the slot into an empty function call. Direct is more efficient here.